Surface_mesh_deformation/test/Surface_mesh_deformation/Cactus_deformation_session_OpenMesh.cpp

```diff
@@ -39,7 +39,7 @@ void compare_mesh(const Mesh& mesh_1, const Mesh& mesh_2)
   {
     total_dif = total_dif + (get(ppmap_1, *it_1) - get(ppmap_2, *it_2));
   }
-  double average_mesh_dif = squared_length(total_dif / num_vertices(mesh_1));
+  double average_mesh_dif = squared_length(total_dif / static_cast<float>(num_vertices(mesh_1)));
 
```

Comment:
This code is strange. `total_diff` is a point, with 3D coordinates. The following should be better:
```C++
const std::size n = num_vertices(mesh_1);
double average_mesh_dif = squared_length(total_dif) / n / n;
```